src/components/AudioTrack/layouts/Row.vue

```diff
@@ -50,8 +50,8 @@
             </span>
           </div>
 
-          <div class="part-b inline-flex space-x-1">
-            <div v-if="isMedium">
+          <div class="part-b inline-flex">
+            <div v-if="isMedium" class="pe-1">
```

Comment:
`space-x-` classes apply to all children as in
```css
.space-x-1 > * + * {
  margin-left: 0.25rem;
}
```
Replacing it with `pe-` classes will mean adding it to all children except the first. Alternatively we could use `gap-` classes on the flex itself?

Reply:
Leaving padding on the last element doesn't seem to introduce any visual problems that I could see. While not being a 1:1 replacement for `space-x-*` I think it's appropriate in this case? I guess the tailwindcss-rtl plugin doesn't cover the `space-<dir>-<unit>`.

`gap` seems like a better solution though. I'll update to that.

Reply:
Fixed!